Sources/StreamChat/Repositories/SyncRepository.swift

```diff
@@ -134,6 +156,20 @@ class SyncRepository {
                 )
             }
         operations.append(contentsOf: refetchChannelListQueryOperations)
+        
+        let channelListQueries = syncQueue.sync {
```

Comment:
We should properly test the sync process, cc @testableapple

Reply:
We tested this quite a bit with @testableapple and found one issue which is fixed now.